pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[tool.poetry]
+name = "supybot-fedora"
+version = "0.0.1"
```

Comment:
Apparently the packaging & release process was done in the Makefile. The version is set there to be `0.3.5`, I think that's what we should use.

Reply:
done!